data/com.github.Alcaro.Flips.desktop

```diff
@@ -0,0 +1,14 @@
+[Desktop Entry]
+Name=Flips
+GenericName=Patcher
+Comment=A patcher for IPS and BPS files
+# Translators: Search terms to find this application. Do NOT translate or localize the semicolons! The list MUST also end with a semicolon!
+Keywords=ips;bps;patcher;
```

Comment:
Keywords are just to search the app, file association can only be done via MIME types, which are listed here: https://gitlab.freedesktop.org/xdg/shared-mime-info/blob/master/freedesktop.org.xml.in.

I couldn't find definitions for IPS and BPS, I'll submit them there, and list them here. It will definitely be better to have *some* support. :)

Reply:
Flips can apply UPS patches too (they start with UPS1), but it's kind of a ninja format - it's not really advertised anywhere. Maybe it too should be added, I'm not sure.

Reply:
Do you know where I can find BPS and UPS patches? It's needed by shared-mime-info for its test suite: https://gitlab.freedesktop.org/xdg/shared-mime-info/merge_requests/53.

Reply:
BPS - well, that's the point of Flips, isn't it? If you want something you didn't create yourself, there are a bunch at RHDN, for example [Der Langrisser translation](https://www.romhacking.net/translations/1147/).

UPS - it's a rare format. I could swear that Langrisser patch was UPS, but not anymore. I have a copy of the UPS, but GitHub won't let me upload it. If it's too much trouble, better ignore it.

Reply:
I found a BPS patch, I'm still looking for some UPS patch though.

Reply:
OK I'll just ignore UPS then. :)